packages/example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
   <application
     android:usesCleartextTraffic="true"
     tools:ignore="GoogleAppIndexingWarning"
-    tools:targetApi="32">
+    tools:targetApi="28">
```

Comment:
[question]: was this causing any issues?

Reply:
Nope